CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 # App Center SDK for iOS, macOS and tvOS Change Log
 
-## Version 2.2.1
+## Version 2.2.1 (Under development)
+
+### App Center Crashes
+
+* **[Fix]** Fix main thread checker's warning during crash processing on macOS.
```

Comment:
To be honest, I'm not exactly sure what is meant by `main thread checker` so I guess this will be confusing for customers, too.

Reply:
Main Thread Checker is a runtime diagnostic provided by Xcode:

<img width="1526" alt="image" src="https://user-images.githubusercontent.com/4802/62437890-75b8ee80-b788-11e9-8033-df2c92217190.png">

I'm pretty sure it's on by default in new projects.

Reply:
Ah got it, I wasn't aware of this setting! Thanks :)